modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -832,6 +820,18 @@ def _done(self, s: ExecutionState, log: StageLog) -> None:
         self.logs.pipeline_stages = _pipeline_stage_parents  # now includes chronology info
         self.logs.materialize(s.log_directory, mode="final")
 
+    # post core pipeline evaluation
```

Comment:
What is "core pipeline evaluation"?

Reply:
Technically, the steps after this comment are still part of the pipeline, so I call the steps prior to that `core pipeline` (the main parts that contains trainings etc.). I'll remove this naming convention and comment though.